test/cli_test.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require_relative '../lib/features/support/interactive_cli'
+
+class InteractiveCLITest < Test::Unit::TestCase
+  def setup
+    @capabilities = { key: 'value' }
+
+    ENV.delete('BRANCH_NAME')
+    ENV.delete('BUILDKITE')
+    ENV.delete('BUILDKITE_PIPELINE_NAME')
+    ENV.delete('BUILDKITE_BUILD_NUMBER')
+    ENV.delete('BUILDKITE_RETRY_COUNT')
+    ENV.delete('BUILDKITE_STEP_KEY')
```

Comment:
Does this need to happen? It doesn't look like `ENV` is used anywhere else in this PR

Reply:
Nope, just a leftover from a copied test setup :)